script/countries.py

```diff
@@ -0,0 +1,32 @@
+"""Helper script to update country list.
+
+ISO does not publish a machine readable list free of charge, so the list is generated
+with help of the pycountry package.
+"""
+import black
+import pycountry
+
+BASE = """
+\"\"\"Automatically generated by currencies.py.
+
+The values are directly corresponding to the ISO 3166 standard. If you need changes
+to the political situation in the world, please contact the ISO 3166 working group.
+
+To update, run python3 -m script.currencies
+\"\"\"
+
+COUNTRIES = {}
+""".strip()
+
+countries = sorted({x.alpha_2 for x in pycountry.countries})
+
+countries_path = "homeassistant/generated/countries.py"
+with open(str(countries_path), "w") as fp:
+    fp.write(
+        black.format_str(
+            BASE.format(
+                ",".join(f'"{x}"' for x in countries),
```

Comment:
😁 
```suggestion
                repr(countries),
```

Reply:
Hmm, but we want to generate a set, not a list.
We could do `repr(countries)[1:-1]` perhaps to drop the enclosing `[ ]`

Reply:
I'd do `.strip("[]")`, but yeah. Or you can add the `set()` call to the `BASE` template :)

Reply:
It did indeed generate an n-tuple, fixed now 👍